changelogs/fragments/664-ipv6-vrf-fix.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - fixes vrf support for ipv6 static routes
```

Comment:
Please update this changelog fragment to follow our guidelines (https://docs.ansible.com/ansible/latest/community/development_process.html#changelog-fragment-entry-format).

Reply:
Hi Kate, I'll update the fragment and tests. Apologies for not doing so (properly) already.. :)